website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -2,73 +2,76 @@
 layout: docs
 page_title: Consul K8s CLI Reference
 description: >-
-  Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
+  The Consul on Kubernetes CLI is a tool for installing and managing Consul on Kubernetes.
 ---
 
-# Consul K8s CLI Reference
+# Consul on Kubernetes CLI Reference
 
-Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes. 
-The Consul K8s CLI allows you to manage the lifecycle of Consul without requiring the usage of `Helm`, [Consul CLI](/commands/index), and `kubectl`. 
-The Consul K8s CLI offers a Kubernetes native experience for managing Consul. 
+The Consul on Kubernetes CLI, `consul-k8s`, is a tool for managing Consul
+without needing `Helm`, the [Consul CLI](/commands/index), or `kubectl`.
```

Comment:
don't you need helm? it's just that the consul-k8s tool will replace the need to directly interact with `helm`


```suggestion
without needing Helm, the [Consul CLI](/commands/index), or `kubectl`.
```

Reply:
That is a good clarification.